src/pages/kb/user-guide/alerts/creating-new-alert-destination.md

```diff
@@ -2,38 +2,65 @@
 category: alerts
 parent_category: user-guide
 helpscout_url: 
-title: Alert Destinations
+title: Adding New Alert Destinations
 slug: creating-new-alert-destination
 order: 3
+toc: true
 ---
 
-Redash supports a vast amount of Alert Destinations.
-The latest list includes: Email, HipChat, Mattermost, PagerDuty, WebHook, Slack, ChatWork.
- 
-![](/assets/images/docs/alerts/alert_destinations_3.png)
+# Intro
 
+Whenever an Alert triggers, it sends a blob of related data (called the Alert Template) to its designated **Alert Destinations**. Destinations can use this blob of data to fire off emails, Slack messages, or custom web hooks. You can set up new Alert Destinations from the settings screen. 
 
-Prior to sending alerts to one of the above destinations, there is a need to define and configure them.
+![](/assets/images/docs/gitbook/create-new-alert-destination.png) 
 
+{% callout warning %}
 
-Adding Alert Destination
+Because custom Alert Destinations are able to send sensitive information to services outside of Redash, only Admins can add new alert destinations. Destinations are available to all users once configured.
```

Comment:
I wouldn't describe it this way, as we're going to allow anyone to create alert destinations. It's just a limitation of the current implementation.